src/screens/Podcast/styles.ts

```diff
@@ -22,6 +22,13 @@ type Styles = {
 }
 
 const styles = StyleSheet.create<Styles>({
+  spinner: {
```

Comment:
We will repeat same styles in every component. Let's put it to Sprinner/styles.ts

Reply:
What if this spinner will be needed in some block? Not for all page

Reply:
We should have default styles, and if we need we can't change it by passing the new styles down as props

Reply:
Thnx! It will be best variant